.github/workflows/release-apps.yml

```diff
@@ -0,0 +1,74 @@
+name: Release Apps
+
+on:
+  release:
+    types: [ published ]
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
```

Comment:
`if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository`

To prevent all workflow from running if it's coming from a forked pull request. Since it can't run without failing as they can't get the access to write into ghcr.io

Reply:
After a little thought I updated the script to use the github org/username as deploy target so that we don't have to worry about hard coded endpoints when people fork the repository.